presto-main/src/test/java/com/facebook/presto/server/TestQuerySessionSupplier.java

```diff
@@ -156,7 +163,27 @@ public void testInvalidTimeZone()
         QuerySessionSupplier sessionSupplier = new QuerySessionSupplier(
                 createTestTransactionManager(),
                 new AllowAllAccessControl(),
-                new SessionPropertyManager());
+                new SessionPropertyManager(),
+                new SqlEnvironmentConfig());
         sessionSupplier.createSession(new QueryId("test_query_id"), context, Optional.empty(), TEST_RESOURCE_GROUP_ID);
     }
+
+    @Test
+    public void testSqlPathCreation()
+    {
+        ImmutableList.Builder<SqlPathElement> correctValues = ImmutableList.builder();
+        correctValues.add(new SqlPathElement("normal", "schema"));
+        correctValues.add(new SqlPathElement("\"who.uses.periods\"", "\"in.schema.names\""));
+        correctValues.add(new SqlPathElement("\"same,deal\"", "\"with,commas\""));
+        correctValues.add(new SqlPathElement("a\"\"terrible", "\"thing!@#$%^&*()\""));
+        List<SqlPathElement> correctList = correctValues.build();
+
+        SqlPath path = new SqlPath("normal.schema,"
+                + "\"who.uses.periods\".\"in.schema.names\","
+                + "\"same,deal\".\"with,commas\","
+                + "a\"\"terrible.\"thing!@#$%^&*()\"");
+        List<SqlPathElement> testList = path.getParsedPath();
+
+        assertEquals(correctList, testList);
```

Comment:
`assertEquals` takes the `actual` result as the first parameter, but the expected result goes first here (the result is the same, but just to be consistent with the API of this method).

Let's rename `correctList` -> `expected` and `testList` -> `actual`, and we can do `assertEquals(actual, expected)`.